nixos/modules/services/x11/window-managers/i3.nix

```diff
@@ -16,6 +16,9 @@ in
       description = lib.mdDoc ''
         Path to the i3 configuration file.
         If left at the default value, $HOME/.i3/config will be used.
+
+        To pass in derivation paths from nix,
```

Comment:
1. Too obfuscated
2. Too paternalistic

Reply:
I was honestly confused as to how to do this when I was learning nix. I was using builtins.toFile because my lsp told me about it but I didnt know about the trivial builders, and was confused as to why I could not include derivation paths. If a comment like this is unwanted, thats ok, but I felt it should be clarified in some way.

Reply:
Your explanation was/is confusing.